test_scripts/API/ButtonSubscription/004_Unsubscribe_Button_SDL_respond_GENERIC_ERROR_timeout.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile App requests SubscribeButton
+-- 2) Mobile App requests UnsubscribeButton
+-- 3) HMI doesn't response.
+-- SDL does:
+-- 1) Respond GENERIC_ERROR to Mobile App after default timeout and not send OnHashChange
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variable ]]
+local buttonName = "OK"
+local errorCode = "GENERIC_ERROR"
+
+--[[ Local function ]]
+local function rpcGenericError()
+    common.rpcHMInotResponce(1, "UnsubscribeButton", buttonName, errorCode)
+    common.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+    :Times(0)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("App registration", common.registerAppWOPTU)
+runner.Step("App activation", common.activateApp)
+runner.Step("Subscribe on " .. buttonName .. " button", common.rpcSuccess, { 1, "SubscribeButton", buttonName })
+
+runner.Title("Test")
+runner.Step("Failure Unsubscribe on "  .. buttonName .. " button, return GENERIC_ERROR", rpcGenericError)
```

Comment:
There is function common.rpcHMInotResponce, so please use it instead if the rpcGenericError